### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by tarasavage**

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const populations = [...document.querySelectorAll('.population')]
+  .map(span => +span
+    .innerHTML
+    .split(',')
+    .join(''));
+
+const total = populations.reduce((x, y) => x + y, 0);
```
Comment (marina-tilniak) [inline src/scripts/main.js:9]:
```suggestion
const total = populations.reduce((sum, population) => x + y, 0);
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_get_data_DOM/)
+    - [DEMO LINK](https://tarasavage.github.io/js_get_data_DOM/)
```
Comment (marina-tilniak) [inline README.md:2]: your link to demo doesn't work, did you do deploy?

**Review by marina-tilniak — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tarasavage) [reply README.md:2]: yes, I don't know why it's not working

Comment (Anastasiia-Svintsova) [reply README.md:2]: I think you forgot to deploy your code. Please, deploy it and add the link to the description
image

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]

**Comment by tarasavage** [comment]
https://tarasavage.github.io/js_get_data_DOM/

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**